packages/SwingSet/test/snapshots/test-state.js.md

```diff
@@ -0,0 +1,15 @@
+# Snapshot report for `test/test-state.js`
+
+The actual snapshot is saved in `test-state.js.snap`.
+
+Generated by [AVA](https://avajs.dev).
+
+## crankhash - initial state and additions
+
+> initial state
+
+    'fb271936c31500468a08241699d29c850bcac0167c4b0bbd0db9d9124926eabe'
+
+> expected activityhash
+
+    '55447e103e9da37a42e37759b09e4c57014a6ccee3273d27af4addac26c28ec1'
```

Comment:
Should this file and the corresponding (binary!) `.snap` file be put in git or is this a commit mistake?

Reply:
Yeah, both belong in git, this is AVA's way to help with the updating of golden-master -type tests. Previously any changes to the initial kernel state (like adding the `vat.nextUpgradeID` key) required `test-state.js` to be modified, to replace the initial activityhash with a new value. Now, you run `yarn test` with an extra argument to say "yep, I know the recorded hash is wrong, please accept the current version as the right one". The `.md` and `.snap` files record the expected version. I'm surprised that `.snap` is binary, but it looks like that's just how AVA does it.

(I'm following the pattern that @turadg established with `test-xsnap-store.js`)